tests/org.eclipse.emfcloud.modelserver.emf.tests/src/org/eclipse/emfcloud/modelserver/emf/common/ModelControllerTest.java

```diff
@@ -78,12 +81,17 @@
    @Mock
    private ServerConfiguration serverConfiguration;
 
+   private Codecs codecs;
+
    private ModelController modelController;
 
    @Before
    public void before() {
       when(serverConfiguration.getWorkspaceRootURI()).thenReturn(URI.createFileURI("/home/modelserver/workspace/"));
-      modelController = new ModelController(modelRepository, sessionController, serverConfiguration);
+      Map<String, Codec> formatsToCodecsMap = new HashMap<>(1);
+      formatsToCodecsMap.put(ModelServerPathParameters.FORMAT_XMI, new XmiCodec());
+      codecs = new Codecs(formatsToCodecsMap);
```

Comment:
minor: Could be simplified to `new Codecs(Map.of(ModelServerPathParameters.FORMAT_XMI, new XmiCodec()))`

Reply:
I'm too much used to J8. Didn't know that one yet. :+1: